classes/class-loader.php

```diff
@@ -110,6 +110,18 @@ private function __construct() {
 
 		// Load block patterns.
 		Block_Pattern::register_all();
+
+		add_action(
+			'wp_enqueue_scripts',
+			static function () {
+				if ( has_block( 'core/media-text' ) ) {
+					wp_enqueue_script(
+						'rellax',
+						'https://cdn.jsdelivr.net/gh/dixonandmoe/rellax@master/rellax.min.js'
```

Comment:
We could use the [version from cdnjs](https://cdnjs.com/libraries/rellax), since we already use it for some libraries here and in the theme (to save one more dns request).

Reply:
[Updated](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/931/commits/30855649e9157ad6cf1f403cd3b4ef4159a3a81b) 🙂